client/src/ui/organisms/placement/index.tsx

```diff
@@ -50,7 +61,7 @@ export function Placement() {
     data: pong,
     isLoading,
     isValidating,
-  } = useSWR<PlacementStatus>(
+  } = useSWR<PlacementStatus | PlacementError>(
```

Comment:
Maybe we can extract `type PlacementData = PlacementStatus | PlacementError` to avoid duplication:

```suggestion
  } = useSWR<PlacementData>(
```